op_tests/triton_tests/test_la.py

```diff
@@ -363,6 +368,8 @@ def main():
     init_dtype = torch.float16
     BLOCK_M = 128
     BLOCK_N = 64
+    XCD_REMAP = True
+    NUM_XCDS = 8
```

Comment:
Please don't hardcode this value here. I just pushed a PR to clean up this
https://github.com/ROCm/aiter/pull/852/files